packages/autocomplete-js/src/__tests__/api.test.ts

```diff
@@ -271,6 +275,29 @@ describe('api', () => {
         ).toBeInTheDocument();
       });
     });
+
+    test('overrides the default translations', () => {
+      const container = document.createElement('div');
+      document.body.appendChild(container);
+
+      const ac = autocomplete<{ label: string }>({
```

Comment:
We can destructure this to not have to think about how to name this variable :p

```suggestion
      const { update } = autocomplete<{ label: string }>({
```

(same applies at other occurrences)

Reply:
Make sense! Updated in https://github.com/algolia/autocomplete/pull/581/commits/2bfc8825fc483bfaaecf9da48d86723d945b5d46